test/models/user_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+class UserTest < ActiveSupport::TestCase
```

Comment:
Also, here, I don't see an explicit test of `followers_who_dont_follow_tags` -- but is this still in progress? Great to see the test shaping up here!